views/templates/hook/product-simulator.tpl

```diff
@@ -7,9 +7,38 @@
  *}
 {if ($pagantisIsEnabled && $pagantisSimulatorIsEnabled)}
     <script>
+        function checkSimulatorContent() {
```

Comment:
We need to set a limit of attemps, maybe we're using lang=it and simulator is not showing. Check if could help you this code: https://github.com/pagantis/woocommerce/blob/master/templates/product_simulator.php#L17

Reply:
Added attempts behavior